pkg/shell/base_index.js

```diff
@@ -362,6 +362,14 @@ var phantom_checkpoint = phantom_checkpoint || function () { };
                     perform_jump(child, control);
                     return;
 
+                } else if (control.command == "logout") {
+                    cockpit.logout(true);
+                    return;
```

Comment:
We should just forward these packets. This makes the assumption that it has exactly the same fields as generated by ```cockpit.logout()```